pandas/_libs/algos.pyx

```diff
@@ -325,7 +324,7 @@ def nancorr(const float64_t[:, :] mat, bint cov=0, minp=None):
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
-def nancorr_spearman(const float64_t[:, :] mat, Py_ssize_t minp=1):
+def nancorr_spearman(const float64_t[:, :] mat, Py_ssize_t minp=1) -> ndarray:
```

Comment:
Does adding these return annotations to the def like this change anything?

Reply:
for functions that arent cdef/cpdef its ornamental